src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
I just copied over the old logic from the old babel relay plugin. I don't have a preference either way.

Side note: don't they do the same thing? ([docs](https://nodejs.org/api/path.html#path_path_resolve_path),  [src](https://github.com/nodejs/node/blob/v6.3.1/lib/path.js#L167-L168))